sopel/coretasks.py

```diff
@@ -1066,6 +1066,7 @@ def auth_proceed(bot, trigger):
             return
         else:
             # Not an expected response from the server
+            LOGGER.info("Aborting SASL: unexpected server reply '%s'" % trigger)
```

Comment:
I was too quick to approve... I think this should be a warning instead.

```suggestion
            LOGGER.warning("Aborting SASL: unexpected server reply '%s'" % trigger)
```

Reply:
Right, I'm gonna apply this to #2190 too (but not push it until I look into fixing that CI failure for py3.3).